pom.xml

```diff
@@ -100,9 +100,9 @@
     <confluent.version>5.3.4</confluent.version>
     <glassfish.version>2.17</glassfish.version>
     <parquet.version>1.10.1</parquet.version>
-    <junit.jupiter.version>5.7.0-M1</junit.jupiter.version>
-    <junit.vintage.version>5.7.0-M1</junit.vintage.version>
-    <junit.platform.version>1.7.0-M1</junit.platform.version>
+    <junit.jupiter.version>5.8.1</junit.jupiter.version>
```

Comment:
the ticket says 5.8.2

Reply:
Yes it will be 5.8.2 here. I tested with 5.8.1 and 5.8.2 respectively. will end up being 5.8.2